haunt-roll-fail/arcs/game-common.scala

```diff
@@ -1105,6 +1156,17 @@ object CommonExpansion extends Expansion {
                 .some(ll)(r => f.at(r).damaged./(u => RepairAction(f, x, r, u, then).as(u.piece.of(f), Image(u.faction.short + "-" + u.piece.name + "-damaged", u.piece.is[Building].?(styles.qbuilding).|(styles.qship)), "in", r)))
                 .cancel
 
+        case RepairsAction(f, x, r, l, then) =>
+            f.pay(x)
+
+            l.foreach { u =>
+                f.damaged :-= u
+            }
+
+            f.log("repaired", l, "in", r, x)
```

Comment:
Can use `l.comma` for commas between ship names :)

Reply:
Fixed.